hands-on/gemv/5-gemv-omp-many-matrices/gemv-omp-many-matrices.f90

```diff
@@ -21,11 +21,11 @@ program gemv_omp
 allocate(Vout(1:Num_calc,1:N),stat=err)
 if(err/=0) print'(a30,i9,i3)', 'ERROR in allocation for Vout',err
   
-A(:,:) = 1.0
-V(:,:) = 1.0
 
 !$omp parallel do
 do i=1,Num_calc
+   A(i,:) = 1.0
+   V(i,:) = 1.0
    call gemv(N,alpha,A(i,:),V(i,:),Vout(i,:))
```

Comment:
Wrong code. Fortran is column major.

Reply:
Case 5,6,7 needs inspection.

Reply:
In case 5,6,7 Arrays have been updated to column major